content/es/working-groups.md

```diff
@@ -0,0 +1,279 @@
+# io.js Grupos de Trabajo
+
+Los Grupos de Trabajo (GT) de io.js son proyectos autónomos creados por el
+[Comité Tecnico (CT)](https://github.com/iojs/io.js/blob/v1.x/GOVERNANCE.md#technical-committee).
+
+Los Grupos de Trabajo pueden formarse en cualquier momento pero deben de ser ratificados por el CT.
+Una vez formados, el trabajo definido en la carta del Grupo de Trabajo es responsabilidad del Grupo de Trabajo en lugar del CT.
```

Comment:
Yes is defined but I left it just in case is the first thing is read. 

The reason for this to happen is that titles are links when translated from markdown to html. You could then share a link that goes directly to that paragraph and could wonder "what is CT"?, then have to search for it on the document. Is annoying for the translator to do this but I think is more helpful for the reader that the other way around.